base/asyncevent.jl

```diff
@@ -72,7 +75,7 @@ mutable struct Timer
         timeout ≥ 0 || throw(ArgumentError("timer cannot have negative timeout of $timeout seconds"))
         interval ≥ 0 || throw(ArgumentError("timer cannot have negative repeat interval of $interval seconds"))
         timeout = UInt64(round(timeout * 1000)) + 1
-        interval = UInt64(round(interval * 1000))
+        interval = UInt64(ceil(interval * 1000))
```

Comment:
Would it make more sense to make sure that `interval` is at least `1` if it is greater than zero on the way in?